src/OAuth1/AbstractProvider.php

```diff
@@ -173,7 +174,7 @@ protected function getToken()
                 $temp, $this->request->get('oauth_token'), $this->request->get('oauth_verifier')
             );
         }
-        $temp = unserialize($this->request->session()->get('oauth_temp'));
+        $temp = unserialize($this->request->session()->get('oauth_temp'), TemporaryCredentials::class);
```

Comment:
Pretty sure this should be:
```suggestion
        $temp = unserialize($this->request->session()->get('oauth_temp'), ['allowed_classes' => [TemporaryCredentials::class]]);
```


https://www.php.net/manual/en/function.unserialize.php